src/resources/views/inc/show_fields.blade.php

```diff
@@ -1,9 +1,14 @@
 {{-- Show the inputs --}}
 @foreach ($fields as $field)
-    <!-- load the view from the application if it exists, otherwise load the one in the package -->
-    @if(view()->exists('vendor.backpack.crud.fields.'.$field['type']))
-        @include('vendor.backpack.crud.fields.'.$field['type'], array('field' => $field))
+    <!-- load the view from type and view_namespace attribute if set -->
+    @if(isset($field['view_namespace']))
+        @include($field['view_namespace'].'.'.$field['type'], ['field' => $field])
     @else
-        @include('crud::fields.'.$field['type'], array('field' => $field))
+        <!-- load the view from the application if it exists, otherwise load the one in the package -->
+        @if(view()->exists('vendor.backpack.crud.fields.'.$field['type']))
```

Comment:
Actually nowadays Laravel checks for existence automatically, and falls back to the package file. I'll remove this old code myself, thank you!